src/index.html

```diff
@@ -7,8 +7,138 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+    <header class="head">
+      <a href="#" class="head__logo"></a>
+      <nav class="head__menu">
+        <ul class="head__menu-container">
+          <li class="head__link">
+            <a href="">Become a host</a>
```

Comment:
не оставляй `href` пустым
```suggestion
            <a href="#">Become a host</a>
```
```suggestion
            <a href="/">Become a host</a>
```